source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,163 @@
+=================
+Change a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to change documents in a MongoDB
+collection using **update** and **replace** operations.
+
+Update operations modify the fields that you specify in one or more
+documents while leaving other fields and values unchanged. Replace
+operations remove all existing fields except for ``_id`` in
+one or more documents and substitute the deleted fields with new fields
+and values.
+
+.. _updateDocuments:
+
+Update
+------
+
+To update one document, use the ``UpdateOne()`` method, and to update
+multiple documents, use the ``UpdateMany()`` method. Each method takes
+an **update document** that includes at least one **update operator**.
+The update operator specifies the type of update to perform. The update
+document also includes the fields and values that describe the change.
+Update documents use the following format:
+
+.. code-block:: go
+
+   bson.D{{"<update operator>", bson.D{{"<field>", "<value>"}, 
+                                       {"<field>", "<value>"}, ... }}, 
+          {"<update operator>", ... }, ... }
+
+See the MongoDB server manual for a :manual:`complete list of update operators
+and descriptions </reference/operator/update-field/>`.
+
+.. note:: Aggregation Pipelines in Update Operations
+
+   If you are using MongoDB Version 4.2 or later, you can use aggregation
+   pipelines made up of a subset of aggregation stages in update operations. For
+   more information on the aggregation stages MongoDB supports in
+   aggregation pipelines, see our tutorial on performing
+   :manual:`updates with aggregation pipelines </tutorial/update-documents-with-aggregation-pipeline/>`.
+
```

Comment:
Suggestion: Before diving into the example, I think it's worth mentioning what happens if the operation is successful -- in this case, it means the document is updated and you can access its [UpdateResult](https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.7.0/mongo#UpdateResult) 

Applies to Replace section too.

Reply:
I'm not sure if that's necessary given the new material I added to address https://github.com/mongodb/docs-golang/pull/14#discussion_r690452455